packages/moonstone/Panels/Panels.js

```diff
@@ -137,12 +147,14 @@ const PanelsBase = kind({
 		className: ({noCloseButton, styler}) => styler.append({
 			hasCloseButton: !noCloseButton
 		}),
-		applicationCloseButton: ({closeButtonBackgroundOpacity, id, noCloseButton, onApplicationClose}) => {
+		exitAppAriaLabel: ({exitAppAriaLabel}) => (exitAppAriaLabel == null) ? $L('Exit app') : exitAppAriaLabel,
```

Comment:
This won't cascade into the `applicationCloseButton` computed prop as you're expected. In order to keep these predictable and optimizable, each computed property receives the same source props and not the results of "previous" computed executions. As a result, in the `applicationCloseButton` computed prop below, `exitAppAriaLabel` would always equal the value from props and not this computed value.

That all said, since you've implemented the same logic within the `ApplicationCloseButton` component, I suggest omitting this logic from this component and letting the `closeButtonAriaLabel` prop flow untouched to `ApplicationCloseButton`.